examples/todo-app/tests/actions.test.ts

```diff
@@ -3,7 +3,9 @@ import { model } from "../src/model";
 
 describe("actions", () => {
   it("adds a todo", async () => {
-    const { dispatch } = model.createStore({
+    const {
+      store: { dispatch },
```

Comment:
Hmm not sure if I like this syntax, it seems too verbose. Would it make sense to return the dispatch at the top level from `createStore`?

Reply:
Definitely possible. Or we can just remove the `Provider` stuff (for now)

Reply:
probably makes sense to keep it in, just moving `dispatch` up for ease-of-access

Reply:
Tried it, but it's a pain with types, so I dropped it.